### ISSUE TITLE
Moneropedia: Remove 'kovri' section in 'Encryption' entry

### ISSUE THREAD

**Issue by erciccione** [monero-project/monero-site#1311]
It's obsolete. To be resolved after #1250 and Norwegian equivalent (at the moment #1305) are merged to avoid conflicts and unnecessary headaches.

**Comment by SamsungGalaxyPlayer** [comment]
For clarity's sake, the task is to remember the reference to kovri in the encryption entry, not the korvi entry correct? The kovri entry should stay imo as a reference (mostly to say we don't plan to ever use it).

**Comment by erciccione** [comment]
> the task is to remember the reference to kovri in the encryption entry, not the korvi entry correct?

Correct. I don't intend to delete the Kovri entry. Sorry if my wording as unclear.
